packages/langium/src/test/langium-test.ts

```diff
@@ -124,6 +125,39 @@ export function expectGoToDefinition(services: LangiumServices, expectEqual: Exp
     };
 }
 
+export interface ExpectedFindReferences extends ExpectedBase {
+    index: number,
```

Comment:
Suggestion: I'm not sure whether having the index in there is actually useful. Let's take the following example:

```
// Note the <|> in this line
interface <|<|>A|> {
  name: string
}
// Note the <|> in this line
ruleA returns <|<|>A|>: name=ID;
```

I believe we should be able to test this in one call, as there's no benefit to calling it like this:

```ts
await findReferences({
  text: grammar,
  index: 0,
  includeDeclaration: true
});
await findReferences({
  text: grammar,
  index: 1,
  includeDeclaration: true
});
```

Since we need to test all of them anyway, we should do this in a single call. Alternatively, you can add a `rangeIndices` property here, which can be used to test multiple references using different calls. That's what we do for the `goto` tests for example:

```ts
await findReferences({
  text: grammar,
  index: 0,
  rangeIndices: [0, 1],
  includeDeclaration: true
});
await findReferences({
  text: grammar,
  index: 1,
  rangeIndices: [2, 3]
  includeDeclaration: true
});
```

However, that would make these tests really convoluted, so I'm just in favor of removing the `index` property.